test/common/setInterval.spec.ts

```diff
@@ -1,6 +1,12 @@
 'use strict';
 import {zoneSymbol} from "../../lib/common/utils";
 
+var isNode = true;
```

Comment:
1) I am not familiar with the web platform or web workers. I can replace this with any check that is desired.
2) Where is the best place for this to live? Should it get its own file?